memory/src/main/scala/filodb.memory/format/vectors/DeltaDeltaVector.scala

```diff
@@ -253,6 +267,8 @@ object DeltaDeltaConstDataReader extends LongVectorDataReader {
       out
     }
   }
+
+  def changes(vector: BinaryVectorPtr, start: Int, end: Int): Int = 0
```

Comment:
This is not always 0.  If the slope is 0, then it is 0; if it is nonzero, then the number of changes for `ConstDDV` is the number of elements (as it is guaranteed that every element changes at the same rate)

Reply:
Feel free to make a test and test different scenarios

Reply:
Used length to get  changes when slope is not 0